### PR TITLE
Add script for re-driving a Snap Application

### PR DESCRIPTION
**Pull request by jessieay**
* bin/re_drive goes through a sign in flow rather than sign up

### CODE REVIEW

**Comment by jessieay** [comment]
[Epic: as a developer (or Alan), want to be able to manually fix a failed drive and retry](https://trello.com/c/BUdIq7KU/345-spike-as-a-non-developer-i-want-to-be-able-to-manually-fix-a-failed-drive-and-retry)

File: spec/features/mi_bridges_driving_spec.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 RSpec.feature "MI Bridges Driving" do
   # to run this in a non-headless way, remove the `:js` flag below
-  scenario "successfully drives a SNAP Application", :js, :driving do
+  scenario "successfully drives, re-drives a SNAP Application", :js, :driving do
```
Comment (jayroh) [inline spec/features/mi_bridges_driving_spec.rb]: `successfully drives, re-drives a SNAP Application`

=> `successfully drives, then re-drives, a SNAP Application`

Something about that initial wording felt a little awkward.

File: lib/mi_bridges/driver/go_back_to_application_page.rb [hunk 2]
```diff
@@ -6,12 +6,18 @@ class GoBackToApplicationPage < BasePage
       def setup; end
 
       def fill_in_required_fields
-        choose " Go back to where you stopped filling out the application."
+        choose_go_back_to_where_left_off
```
Comment (jayroh) [inline lib/mi_bridges/driver/go_back_to_application_page.rb]: yaaaas

File: spec/features/mi_bridges_driving_spec.rb (hunk 1 above)
Comment (jessieay) [reply spec/features/mi_bridges_driving_spec.rb]: good call, update at 42ef937

File: lib/mi_bridges/driver/go_back_to_application_page.rb (hunk 2 above)
Comment (jessieay) [reply lib/mi_bridges/driver/go_back_to_application_page.rb]: interestingly, choosing the label did not work but choosing that random ID does 🤦‍♂️

**Pull request merged**